src/plugins/controls/public/control_group/editor/control_editor.tsx

```diff
@@ -99,6 +100,10 @@ export const ControlEditor = ({
     dataViews: { getIdsWithTitle, getDefaultId, get },
     controls: { getControlFactory },
   } = pluginServices.getServices();
+
+  const { useEmbeddableSelector: select } = useControlGroupContainerContext();
+  const editorConfig = select((state) => state.componentState.editorConfig);
```

Comment:
Don't need to change this in this PR, but we should probably structure all the different `hide<...>` options list settings the same way... i.e. something like:
```typescript
optionsListConfig: { 
     hideActionBar?: boolean,
     hideExclude?: boolean,
     ...
}
```
I think that would be a whole lot cleaner than it currently is :+1:

Reply:
That's a good call - likely we should change that! Especially because it's only used via code. Might do that in a follow--up!